multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/TimeoutAsyncFlowableStep.java

```diff
@@ -57,13 +56,11 @@ private String getStepName() {
 
     public abstract Duration getTimeout(ProcessContext context);
 
-    protected Duration calculateTimeout(ProcessContext context, TimeoutType timeoutType,
-                                        Variable<Duration> timeoutCommandLineLevelParameter,
-                                        Variable<Duration> timeoutGlobalLevelParameter) {
+    protected Duration calculateTimeout(ProcessContext context, TimeoutType timeoutType) {
         DeploymentDescriptor descriptor = context.getVariable(Variables.DEPLOYMENT_DESCRIPTOR);
         CloudApplicationExtended app = context.getVariable(Variables.APP_TO_PROCESS);
         Integer timeoutModuleLevelValue = extractTimeoutFromAppAttributes(app, timeoutType.getModuleLevelParamName());
```

Comment:
maybe move this under the first 'else' statement so that we don't search for the value unless we need to